lms/djangoapps/shoppingcart/models.py

```diff
@@ -1761,7 +1755,7 @@ def add_to_order(cls, order, course_id, qty, mode_slug=CourseMode.DEFAULT_SHOPPI
         item.list_price = cost
         item.qty = qty
         item.line_desc = _(u'Enrollment codes for Course: {course_name}').format(
-            course_name=course.display_name_with_default_escaped)
+            course_name=course.display_name_with_default)
```

Comment:
These line_desc entries get written to shoppingcart_orderitem, and empirically it doesn't seem that any display_name values to date have contained &lt; or &gt; (which seem to be the extent of the escaping).